custom/conf/app.ini.sample

```diff
@@ -141,8 +141,9 @@ KEYWORDS = go,git,self-hosted,gitea
 [markdown]
 ; Enable hard line break extension
 ENABLE_HARD_LINE_BREAK = false
-; List of custom URL-Schemes that are allowed as links when rendering Markdown
-; for example git,magnet
+; Comma separated list of custom URL-Schemes that are allowed as links when rendering Markdown
+; for example http,https,git,magnet,ftp (more at https://en.wikipedia.org/wiki/List_of_URI_schemes)
+: The default is http and https only.
```

Comment:
http and https are always allowed; typing in a list without them will not disallow them. Please change the comment to reflect this better.